master/buildbot/statistics/storage_backends/influxdb_client.py

```diff
@@ -42,8 +42,9 @@ def __init__(self, url, port, user, password, db, captures,
         self.name = name
 
         self.captures = captures
-        self.client = InfluxDBClient(self.url, self.port, self.user,
-                                     self.password, self.db)
+        if InfluxDBClient is not None:
```

Comment:
I think we should rather do config error in case of InfluxDBClient not installed